frontend/components/CourseOverviewCard/CourseOverviewCard.tsx

```diff
@@ -14,7 +14,7 @@ import {SessionBlock} from '@/components/SessionBlock/SessionBlock';
 
 export declare interface CourseSessionChipProps {
     /** the session of the course */
-    session: "Fall"
+    session: string
```

Comment:
Want specific strings, for example 
```suggestion
    session: "Fall" | "Winter" | "Summer" 
```